src/OWML.ModLoader/Owo.cs

```diff
@@ -193,6 +193,17 @@ private IModBehaviour InitializeMod(Type modType, IModHelper helper)
 			{
 				var mod = _goHelper.CreateAndAdd<IModBehaviour>(modType, helper.Manifest.UniqueName);
 				_console.WriteLine("Added! Initializing...", MessageType.Debug);
+
+				if (mod == default)
```

Comment:
not `== null`?

Reply:
default is a better choice when dealing with generics

Reply:
the default value of any reference type is null

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values